app/controllers/account/chrome_extension_controller.rb

```diff
@@ -0,0 +1,26 @@
+class Account::ChromeExtensionController < ApplicationController
+  before_action :authenticate_user!
+
+  def new
+    @page = Page.new(source_url: params[:source_url],
```

Comment:
looks ugly,
mb some of the way `Page.new(params.permit(:title, :source_url)` ?

Reply:
permit is not needed here. we are passing default values to form, so it's ok